src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Process_StreamTests.cs

```diff
@@ -9,105 +9,112 @@ namespace System.Diagnostics.ProcessTests
+        public void Process_SyncErrorStream()
         {
             Process p = CreateProcessError();
             p.StartInfo.RedirectStandardError = true;
             p.Start();
-            if (!p.HasExited)
-            {
-                string s = p.StandardError.ReadToEnd();
-                p.WaitForExit();
-
-                s = s.Replace("\r", "").Replace("\n", "");
-
-                Assert.Equal("Unhandled Exception: Intentional Exception thrown", s);
-            }
+            Assert.Equal(p.StandardError.ReadToEnd(), "ProcessTest_ConsoleApp.exe error stream\r\n");
+            p.WaitForExit();
         }
 
-        private static StringBuilder s_process_AsyncErrorStream_sb = new StringBuilder();
+        private StringBuilder process_AsyncErrorStream_sb = new StringBuilder();
 
         [Fact]
-        public static void Process_AsyncErrorStream()
+        public void Process_AsyncErrorStream()
         {
-            s_process_AsyncErrorStream_sb.Clear();
+            process_AsyncErrorStream_sb.Clear();
             Process p = CreateProcessError();
             p.StartInfo.RedirectStandardError = true;
             p.ErrorDataReceived += process_AsyncErrorStream_ErrorDataReceived;
             p.Start();
-            if (!p.HasExited)
-            {
-                p.BeginErrorReadLine();
-                p.WaitForExit();
-                Assert.Equal("Unhandled Exception: Intentional Exception thrown", s_process_AsyncErrorStream_sb.ToString());
-            }
+            p.BeginErrorReadLine();
+            p.WaitForExit();
+            Assert.Equal("ProcessTest_ConsoleApp.exe error stream", process_AsyncErrorStream_sb.ToString());
         }
 
-        public static void process_AsyncErrorStream_ErrorDataReceived(object sender, DataReceivedEventArgs e)
+        public void process_AsyncErrorStream_ErrorDataReceived(object sender, DataReceivedEventArgs e)
         {
-            s_process_AsyncErrorStream_sb.Append(e.Data);
+            process_AsyncErrorStream_sb.Append(e.Data);
         }
 
         [Fact]
-        public static void Process_SyncOutputStream()
+        public void Process_SyncOutputStream()
         {
-            Process p = CreateProcess();
+            Process p = CreateProcessStream();
             p.StartInfo.RedirectStandardOutput = true;
             p.Start();
-            if (!p.HasExited)
-            {
-                string s = p.StandardOutput.ReadToEnd();
-                p.WaitForExit();
-                Assert.Equal(s, "ProcessTest_ConsoleApp.exe started\r\nProcessTest_ConsoleApp.exe closed\r\n");
-            }
+            string s = p.StandardOutput.ReadToEnd();
+            p.WaitForExit();
+            Assert.Equal(s, "ProcessTest_ConsoleApp.exe started\r\nProcessTest_ConsoleApp.exe closed\r\n");
         }
 
-        private static StringBuilder s_process_AsyncOutputStream_sb = new StringBuilder();
+        private StringBuilder process_AsyncOutputStream_sb = new StringBuilder();
```

Comment:
I have changed these to use lambda expressions, However personally I prefer to use lambda expressions only for single statement method bodies.